ops/storage-ddls/cassandra-setup.cql

```diff
@@ -208,3 +208,33 @@ WITH OPTIONS = {
 'tokenization_enable_stemming': 'true',
 'tokenization_normalize_lowercase': 'true'
 };
+
+DROP TABLE IF EXISTS timeseriestrends;
+CREATE TABLE timeseriestrends(
+    periodstartdate timestamp,
+    periodtype text,
+    period text,
+    tilez int,
+    tilex int,
+    tiley int,
+    trendscore float,
+    insertion_time timestamp,
+    topic text,
+    PRIMARY KEY ((periodtype, topic, tilez), periodstartdate, period, tilex, tiley)
```

Comment:
Hi @c-w sorry for the delay - your question made me realize I have to tweak this a bit. So, stay tuned for a change. In theory, we need to perform a query like this:

```
select * timeseriestrends where periodtype=? and period in ? and tilez=? and (tilex, tiley) >= ? and (tilex, tiley) < ? order by trendscore; 
```